packages/react-components/src/components/styles/TypingIndicator.styles.ts

```diff
@@ -4,25 +4,16 @@
 import { mergeStyles } from '@fluentui/react';
 
 export const typingIndicatorContainerStyle = mergeStyles({
-  height: '2.125rem',
-  display: 'flex',
-  alignItems: 'center'
-});
-
-export const typingIndicatorListStyle = mergeStyles({
-  fontWeight: 400,
-  whiteSpace: 'nowrap',
-  overflow: 'hidden',
-  textOverflow: 'ellipsis',
-  paddingRight: '0.1875rem'
+  height: '2.125rem'
 });
 
-export const typingIndicatorVerbStyle = mergeStyles({
+export const typingIndicatorStringStyle = mergeStyles({
   fontWeight: 400,
-  whiteSpace: 'nowrap',
+  whiteSpace: 'pre',
```

Comment:
When does this wrap vs use the ellipses? (or whats the purpose of `pre` here?)

Reply:
Removed the styling regarding wrapping. I changed it to `pre` so that whitespace in the span elements are not ignored.
`whiteSpace: nowrap`
![image](https://user-images.githubusercontent.com/79475487/123353957-83f88b00-d517-11eb-8717-af75e46ad86b.png)
vs.
`whiteSpace: pre`
![image](https://user-images.githubusercontent.com/79475487/123354009-9a9ee200-d517-11eb-99f4-15bbae4a6e8c.png)

Reply:
Ah thanks!